pajbot/apiwrappers/base.py

```diff
@@ -73,25 +73,34 @@ def join_base_and_endpoint(base, endpoint):
         else:
             return BaseAPI.join_base_and_string(base, endpoint)
 
-    def request(self, method, endpoint, params, headers, json=None, **request_options):
```

Comment:
I strongly disagree with these parameters being added

The api base class already does this via the "endpoint" parameter